src/jit/emitxarch.cpp

```diff
@@ -2059,6 +2059,8 @@ UNATIVE_OFFSET emitter::emitInsSizeAM(instrDesc* id, code_t code)
         size += emitGetRexPrefixSize(ins);
     }
 
+    size += emitAdjustSizeCrc32(ins, attrSize);
```

Comment:
What about the `emitInsSizeSV` and `emitInsSizeRR` cases?

Reply:
They work fine now. I carefully built the regression test that has covered all the code size cases for CRC32.

Reply:
Oh, yes, I just found CRC32-specific path in RS. Will update, thank you!

Reply:
Done.